go.mod

```diff
@@ -48,8 +48,11 @@ require (
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.8.0
 	github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azsecrets v1.3.0
+	github.com/csi-addons/kubernetes-csi-addons v0.9.0
 )
 
+require github.com/go-task/slim-sprig/v3 v3.0.0 // indirect
```

Comment:
we already have require section for indirect  dep, this should be added there, please check